request_handler.py

```diff
@@ -102,14 +110,13 @@ def do_POST(self):
         if resource == 'posts':
             response = create_post(post_body)
         if resource == 'comments':
-            response = create_comment(post_body)
+            new_comment = create_comment(post_body)
+            self.wfile.write(f"{new_comment}".encode())
```

Comment:
This line needs to be placed around 111 like you had it.  It's unneccassry to have it scoped just for comments. Consider using the variable `response` created in line 96 to refactor the code just as line 111 suggests